### PR TITLE
Add integrations queries for Flint

### PR DESCRIPTION
**Pull request by Swiddis**
For making integrations work with Flint, we need to provide a set of queries that can be used to create OpenSearch indices for querying. The PR is still in the draft phase as we compile our queries and work towards testing them in real scenarios, and adding unit tests + documentation for the functionality.

Todo:
- [X] ELB Table Queries
- [ ] ~~Nginx Table Queries~~
- [ ] ~~Get VPC Flow queries from @YANG-DB~~
- [ ] ~~Get Cloudtrail queries from @YANG-DB (available in https://github.com/opensearch-project/opensearch-catalog/pull/81)~~
- [x] Add queries to create aliased MV indices from tables
- [x] Add aliasing logic in frontend
- [ ] Testing
- [x] Bugfixing

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by YANG-DB — APPROVED** [review]
@Swiddis 
thanks for all the work 
can u also update the appropriate assets in the corresponding catalog integration PR ?

**Review by YANG-DB — APPROVED** [review]

File: public/components/integrations/components/setup_integration.tsx [hunk 1]
```diff
@@ -60,9 +61,9 @@ const INTEGRATION_CONNECTION_DATA_SOURCE_TYPES: Map<
   [
     's3',
     {
-      title: 'Table',
-      lower: 'table',
-      help: 'Select a table to pull the data from.',
+      title: 'Catalog',
```
Comment (derek-ho) [inline public/components/integrations/components/setup_integration.tsx:64]: Why catalog here?

File: public/components/integrations/components/setup_integration.tsx [hunk 2]
```diff
@@ -185,6 +188,7 @@ export function SetupIntegrationForm({
     [] as Array<{ label: string }>
   );
   const [isSuggestionsLoading, setIsSuggestionsLoading] = useState(true);
+  const [isBlurred, setIsBlurred] = useState(false);
```
Comment (derek-ho) [inline public/components/integrations/components/setup_integration.tsx:191]: there is also onBlur function that comes along with most elements that you can use and avoid this separate re-implementation

Comment (Swiddis) [reply public/components/integrations/components/setup_integration.tsx:191]: Where is it documented? I'd heard about it but didn't find any references.

File: public/components/integrations/components/setup_integration.tsx (hunk 1 above)
Comment (Swiddis) [reply public/components/integrations/components/setup_integration.tsx:64]: Because the form isn't referring to a table, I asked around for a better name.

**Review by derek-ho — APPROVED** [review]

**Pull request merged**